tests/integration/src/test/resources/org/mule/test/integration/exceptions/exception-dlq-flow.xml

```diff
@@ -14,8 +14,6 @@
     <jms:activemq-connector name="jmsConn" maxRedelivery="1"
         brokerURL="vm://localhost?broker.persistent=false&amp;broker.useJmx=false"/>
 
-    <jboss-transaction-manager/>
```

Comment:
This tx manager was not being initialized for this test. Now that the init is eager, it was failing because of a JNDI lookup.